lib/scss/stanford_events.views.scss

```diff
@@ -0,0 +1,178 @@
+@charset 'UTF-8';
+@import "./stanford_events.config";
+
+// Filtered list
+.stanford-event-terms-utility-results-for {
+  @include modular-spacing('margin', null null 2 null);
+
+  > div,
+  h2 {
+    @include small-paragraph;
+    display: inline-block;
+
+  }
+}
+
+.su-event-list-item {
```

Comment:
This whole selector of css should be separated out into its own scss/css files and included through the events-list.ui_pattern.yml settings.

Reply:
@jenbreese see above comment.

Reply:
@jenbreese please move this into its own css file and attach to the pattern.